lib/os/Kconfig

```diff
@@ -161,6 +163,7 @@ endif
 
 config REBOOT
 	bool "Reboot functionality"
+	imply CACHE_MANAGEMENT
```

Comment:
Ditto.  Platforms that can reboot and platforms with cache aren't the same set.  If a platform has reboot code that requires cache, then the platform should be selecting this, no?